docs/src/man/getting_started.md

```diff
@@ -15,17 +15,16 @@ relevant variables into your current namespace.
 
 !!! note
 
-    By default Jupyter Notebook will limit the number of rows and columns when displaying a data frame to roughly
-    fit the screen size (like in the REPL).
-
-    You can override this behavior by changing the values of the `ENV["COLUMNS"]` and `ENV["LINES"]`
-    variables to hold the maximum width and height of output in characters respectively.
+    By default DataFrames.jl limits the number of rows and columns when displaying a data frame in a Jupyter
```

Comment:
```suggestion
    By default DataFrames.jl limits the number of rows and character columns when displaying a data frame in a Jupyter
```

Reply:
In fact, it is really data columns. We do not track the number of printed characters as we do in text backends. Thus, `DATAFRAME_COLUMNS`  contains how many columns will be rendered.

Reply:
Ah - OK. So this is a change.
@nalimilan - in that case do you think 500 is too much or OK?

Reply:
Btw, I think 500 is too much :) I did some tests and it is a lot of columns.

Reply:
Let us decrease it to 100. I will add appropriate change propositions.